modules/n1ql/pages/n1ql-language-reference/delete.adoc

```diff
@@ -172,7 +192,7 @@ include::ROOT:partial$query-context.adoc[tag=section]
 
 WARNING: Please note that running the following examples will permanently delete your sample data.
 To restore your sample data, remove and reinstall the `travel-sample` bucket.
-Refer to xref:manage:manage-settings/install-sample-buckets.adoc[Sample Buckets] for details. 
+Refer to xref:server:manage:manage-settings/install-sample-buckets.adoc[Sample Buckets] for details. 
```

Comment:
In this case I am sure this change is unnecessary. When we merge this change into Capella, we would have to update this xref anyway, to point to the Capella instructions for installing example files. So adding the server component here is pointless.